CHANGELOG.next.toml

```diff
@@ -162,3 +162,9 @@ FIPS and dual-stack endpoints can each be configured in multiple ways:
 Note: Not all services support FIPS and dual-stack.
 """
 author = "rcoh"
+
+[[smithy-rs]]
+message = "Add serde support for datetime."
```

Comment:
Would be good to indicate that this is an experimental feature, and demonstrate how to enable it via the compiler flag and features.

Alternatively, you could hold off on adding a changelog entry until all the serde implementation is done and then have one larger changelog entry that describes all of it.

Reply:
I thought each PR has to have an entry.

I think I will create a one big CHANGELOG once it's done.

Reply:
> I thought each PR has to have an entry.

There is no such rule 😄 

Each changelog toml entry becomes a line in the release notes, so we generally try to make that information as helpful as possible to folks that are upgrading to the latest version of the SDK.

Reply:
reverted